benchmarks/bm/iast_fixtures/str_methods.py

```diff
@@ -1015,73 +1003,73 @@ def do_slice_condition(s: str, first, second):
     return s[first : second or 0]
 
 
-def do_namedtuple(s: Text) -> Any:
+def do_namedtuple(s: Text):
     PathInfo = namedtuple("PathInfo", "name surname")
     my_string = PathInfo(name=s, surname=None)
     return my_string
 
 
-def do_split_no_args(s):  # type: (str) -> List[str]
+def do_split_no_args(s):
     return s.split()
 
 
-def do_rsplit_no_args(s):  # type: (str) -> List[str]
+def do_rsplit_no_args(s):
     return s.rsplit()
 
 
-def do_split_maxsplit(s, maxsplit=-1):  # type: (str, int) -> List[str]
+def do_split_maxsplit(s, maxsplit=-1):
```

Comment:
type missing or partially missing on all functions from here to the end of the file